react-native-sdk/update_dependencies.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+
+const packageJSON = require('../../YOUR_APP/package.json');
```

Comment:
You don't need to set YOUR_APP there, since you know you'll be 2 levels down, so `../../package.json` should point to the app's file.